cmd/rekor_monitor/main.go

```diff
@@ -126,6 +188,9 @@ func main() {
 
 	// To get an immediate first tick, for-select is at the end of the loop
 	for {
+		if runAsMonitor != nil && *runAsMonitor {
```

Comment:
I think we can unconditionally increment metrics since these are global variables and always initialized.